modules/ROOT/pages/update-acm.adoc

```diff
@@ -80,3 +98,7 @@ When the upgrade finishes, you receive a notification email.
 ... Ensure that object level permissions include only `Create` and `Read`.
 ... Ensure that field level permissions for all fields include only `Read` and `Edit`.
 .. Check that your permissions match those in the xref:create-community.adoc#configure-profiles-and-permissions[Configure Profiles and Permissions] documentation.
+. If you are upgrading from a package version before 1.74, follow these steps:
```

Comment:
Before 1.7.4 inclusive of the aforementioned "before 1.6.9"?

Reply:
Yes, customers upgrading from a version before 1.69 will follow both the "before 1.69" steps and the "before 1.74" steps.